### ISSUE TITLE
is this project still under development?

### ISSUE THREAD

**Issue by nnako** [alex-LE/yourTinyTodo#93]
Hi,

I like the simple way of handling todo lists...

Is this project still under development? The example links are not working...

Thanks.

**Comment by jniggemann** [comment]
No, not really. But if we find more developers this may change.

**Comment by nnako** [comment]
great. I saw that the this project was based on an old version of Max Pozdeev's [mytinytodo](http://www.mytinytodo.net/) and that there are some other forks like from [binfalse](https://github.com/binfalse/MyTinyTodo) and [drahmel](https://github.com/drahmel/myTinyTodo). But all of them (and even yourTinyTodo, here) seem to have lost track of its base, as myTinyTodo was updated to v1.4.3 in early 2014 and changes of all other projects mentioned here date back to 2013.

What exactly was missed in "myTinyTodo" that made "yourTinyTodo" evolve? Which features did the base project not provide? Are there any plans for "yourTinyTodo"? At which part would this project need some further development?

Thanks.
